### ISSUE TITLE
UserWarning: size_average and reduce args will be deprecated, please use reduction='sum' instead.

### ISSUE THREAD

**Issue by gorkem7** [qfgaohao/pytorch-ssd#100]
I am having this message "UserWarning: size_average and reduce args will be deprecated, please use reduction='sum' instead." and then this: "Segmentation fault:11" 

is there any idea what is the problem? 
I believe UserWarning does not affect training progress but segmentation fault. 

Here is the full message with my run code;

python train_ssd.py --dataset_type voc  --datasets /Users/gorkem/Development/master-thesis-gorkem__/VOC2012/train  --validation_dataset /Users/gorkem/Development/master-thesis-gorkem__/VOC2012/test --net mb2-ssd-lite --base_net models/mb2-imagenet-71_8.pth  --scheduler cosine --lr 0.01 --t_max 200 --validation_epochs 5 --num_epochs 20

2020-02-18 15:24:30,619 - root - INFO - Namespace(balance_data=False, base_net='models/mb2-imagenet-71_8.pth', base_net_lr=None, batch_size=32, checkpoint_folder='models/', dataset_type='voc', datasets=['/Users/gorkem/Development/master-thesis-gorkem__/VOC2012/train'], debug_steps=100, extra_layers_lr=None, freeze_base_net=False, freeze_net=False, gamma=0.1, lr=0.01, mb2_width_mult=1.0, milestones='80,100', momentum=0.9, net='mb2-ssd-lite', num_epochs=20, num_workers=4, pretrained_ssd=None, resume=None, scheduler='cosine', t_max=200.0, use_cuda=True, validation_dataset='/Users/gorkem/Development/master-thesis-gorkem__/VOC2012/test', validation_epochs=5, weight_decay=0.0005)
2020-02-18 15:24:30,620 - root - INFO - Prepare training datasets.
2020-02-18 15:24:30,623 - root - INFO - No labels file, using default VOC classes.
2020-02-18 15:24:30,624 - root - INFO - Stored labels into file models/voc-model-labels.txt.
2020-02-18 15:24:30,624 - root - INFO - Train dataset size: 11540
2020-02-18 15:24:30,624 - root - INFO - Prepare Validation datasets.
2020-02-18 15:24:30,626 - root - INFO - No labels file, using default VOC classes.
2020-02-18 15:24:30,626 - root - INFO - validation dataset size: 10991
2020-02-18 15:24:30,626 - root - INFO - Build network.
2020-02-18 15:24:30,693 - root - INFO - Init from base net models/mb2-imagenet-71_8.pth
2020-02-18 15:24:30,721 - root - INFO - Took 0.03 seconds to load the model.
2020-02-18 15:24:30,724 - root - INFO - Learning rate: 0.01, Base net learning rate: 0.01, Extra Layers learning rate: 0.01.
2020-02-18 15:24:30,724 - root - INFO - Uses CosineAnnealingLR scheduler.
2020-02-18 15:24:30,724 - root - INFO - Start training from epoch 0.
/opt/anaconda3/lib/python3.7/site-packages/torch/nn/_reduction.py:46: UserWarning: size_average and reduce args will be deprecated, please use reduction='sum' instead.
  warnings.warn(warning.format(ret))
Segmentation fault: 11

Using Pytorch 1.1.0

**Comment by gorkem7** [comment]
I have downgraded the Pytorch to 1.0, and problem solved.

**Comment by sirvnvu98** [comment]
I used Pytorch = 1.0 but still got the warning: G: \ Anaconda3 \ lib \ site-packages \ torch \ nn \ _reduction.py: 49: UserWarning: size_average and reduce args will be deprecated, please use reduction = 'sum 'instead.warnings.warn (warning.format (ret))
